src/indicators/spinner.js

```diff
@@ -26,12 +27,15 @@ import { filterInvalidDOMProps } from '../utils'
  *
  */
 
-const propTypes = {}
+const propTypes = {
+  className: ''
```

Comment:
This isn't accurate. In `propTypes` you use `PropTypes` to declare the type of the prop. In `defaultProps`, you declare the default value for the properties that aren't required in `propTypes`

Reply:
Here's an example: https://github.com/LaunchPadLab/lp-components/blob/ead699069fa1a2c322c7a117b3c2c6a196c2b1c1/src/forms/inputs/input.js#L37-L45